public/projects/pirate_translator.md

```diff
@@ -0,0 +1,27 @@
+# Pirate Translator
+
+A simple command-line or single-page web toy that takes plain English text and "translates" it into the pirate dialect.
+
+The pirate dialect consists mostly of emphasizing "R", replacing or abbreviating some words ("hello" => "ahoy"), and inserting colorful idioms like "Avast!" and "Shiver me timbers!" between sentences. 
+
+
+## Tech
+
+* parsing
+* strings
+* [random number generation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)
+
+## References
+
+* http://pirate.monkeyness.com/online_pirate_translator
+* http://speakpirate.com/
+
+## Features
+
+* preserve case
+* fixed translations
+* randomized insertions
+
+## Todo: full test-driven suite?
```

Comment:
Is the translator deterministic? It seems that each of the reference translators result in different outputs.

Maybe we have an introduction where the project explains the rules of speaking piratese?